src/services/BadgesService.ts

```diff
@@ -52,4 +55,52 @@ export class BadgesService {
   private static getIpfsHttpsLink(ipfsLink: string) {
     return ipfsLink.replace('ipfs://', 'https://ipfs.io/ipfs/')
   }
+
+  public static async grantBadgeToUsers(badgeCid: string, users: any[]): Promise<string> {
+    const badgeOwners = await OtterspaceSubgraph.get().getBadgeOwners(badgeCid)
+
+    const usersWithoutBadge = users.filter((user) => {
+      return !badgeOwners.includes(user.toLowerCase())
+    })
+
+    if (usersWithoutBadge.length > 0) {
+      return await this.airdropWithRetry(badgeCid, usersWithoutBadge)
+    } else {
+      return 'All recipients already have this badge'
```

Comment:
Should this throw Error?

Reply:
I don't think it's an error, but rather that all specified users have that badge. 
(The message is for us in the debug section)

I'd leave the errors for things that went wrong during the airdrop